views/user/story-edit.hbs

```diff
@@ -1,13 +1,19 @@
 <h2>Edit the story</h2>
 
 <form action="/stories/{{story.id}}/edit" method="POST">
-<div class="textarea">
-  <label for="title"> Title: </label>
+  <div class="textarea">
+    <label for="title"> Title: </label>
```

Comment:
Remove the useless whitespace  in between the label  `<label for="title"> Title: </label>`